src/Psr11ConnectionRegistry.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Doctrine\Bundle\DBALBundle;
+
+use Doctrine\DBAL\Connection;
+use Psr\Container\ContainerInterface;
+
+class Psr11ConnectionRegistry implements ConnectionRegistry
```

Comment:
@stof is this what you had in mind for the registry? I would first make it work nicely here on the bundle and then add a PR to move it to `doctrine/dbal`

Reply:
looks good to me.